recipes/enjincppsdk/all/conanfile.py

```diff
@@ -0,0 +1,139 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.43.0"
+
+
+class EnjinCppSdk(ConanFile):
+    name = "enjincppsdk"
+    description = "A C++ SDK for development on the Enjin blockchain platform."
+    license = "Apache-2.0"
+    topics = ("enjin", "sdk", "blockchain")
+    homepage = "https://github.com/enjin/enjin-cpp-sdk"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    settings = "os", "compiler", "arch", "build_type"
+    generators = "cmake", "cmake_find_package", "cmake_find_package_multi"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_default_http_client": [True, False],
+        "with_default_ws_client": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": False,
```

Comment:
```suggestion
        "fPIC": True,
```

Keep fPIC True by default